### PR TITLE
📃 added docs for Syncing An Old Forked Repository

### PR DESCRIPTION
**Pull request by ZibanPirate**
Added documentation on how to sync and old-forked repo with the original one

## Checklist

- [ ] `yarn test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](https://github.com/dzcode-io/dzcode.io/blob/master/.github/CONTRIBUTING.md#coding-guidelines)
- [x] Affected `data`
- [ ] Affected `frontend`

### CODE REVIEW

**Review by Souhaib-Benbouzid — APPROVED** [review]
amazing article.

File: data/documentation/Git_Basics/Syncing_An_Old_Forked_Repository/info.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "title": "Syncing An Old Forked Repository",
```
Comment (NurElHuda) [inline data/documentation/Git_Basics/Syncing_An_Old_Forked_Repository/info.json:2]: The title is a little vague. Synch with what?

Comment (ZibanPirate) [reply data/documentation/Git_Basics/Syncing_An_Old_Forked_Repository/info.json:2]: Hmm, what do u suggest as a title?

Comment (ZibanPirate) [reply data/documentation/Git_Basics/Syncing_An_Old_Forked_Repository/info.json:2]: i just wanted to keep it short for the sidebar and social-media thumbnails

Comment (NurElHuda) [reply data/documentation/Git_Basics/Syncing_An_Old_Forked_Repository/info.json:2]: "Syncing An Old Forked Repository With Upstream" ?

**Review by NurElHuda — APPROVED** [review]
Great article.

**Review by NurElHuda — COMMENTED** [review]
Everything looks good.

**Pull request merged**